UnityProject/Assets/Scripts/Systems/StationObjectives/StationObjectiveManager.cs

```diff
@@ -81,11 +81,18 @@ private string GetObjectiveStatus()
 
 		private string GetObjectiveStatusNonRich()
 		{
-			var message = $"Objective of {MatrixManager.MainStationMatrix.GameObject.scene.name}:\n";
-			var complete = activeObjective.CheckCompletion();
-			message += $"{activeObjective.GetRoundEndReport()}";
-			message += complete ? "Completed\n" : "Failed\n";
-			return message;
+			var message = $"Status of {MatrixManager.MainStationMatrix.GameObject.scene.name}:\n";
+			if (activeObjective != null)
+			{
+
+				bool complete = false;
+
+				complete = activeObjective.CheckCompletion();
+				message += $"{activeObjective.GetRoundEndReport()}";
+				message += complete ? "Completed\n" : "Failed\n";
+				return message;
+			}
```

Comment:
```suggestion
			var message = $"Status of {MatrixManager.MainStationMatrix.GameObject.scene.name}:\n";
			if (activeObjective != null)
			{
				message += $"{activeObjective.GetRoundEndReport()}";
				message += activeObjective.CheckCompletion() ? "Completed\n" : "Failed\n";
				return message;
			}
```

Codacy gonna complain about string builder too

Reply:
Oh wait sorry you only added the null check, not the complete var stuff.

Reply:
Resolve?

Reply:
yeah